dis_snek/api/http/route.py

```diff
@@ -1,27 +1,54 @@
 from typing import TYPE_CHECKING, Any, ClassVar, Optional
 from urllib.parse import quote as _uriquote
 
+
 if TYPE_CHECKING:
     from dis_snek.models.discord.snowflake import Snowflake_Type
 
 
 class Route:
```

Comment:
This should probably also implement `__eq__()`, I've made that mistake myself..

Reply:
Why would you need to check equality of a Route?

Reply:
You made it hashable, if you don't implement `__eq__()` then a Route can't be stored in a dictionary (well it can, but it falls back to `object()`'s implementation of `is` checks).

Reply:
Very valid point, i'll add that